src/Config/Configuration.php

```diff
@@ -101,4 +91,14 @@ public function getFlagETag(): ?string
     {
         return $this->flagsConfig?->eTag ?? null;
     }
+
+    public function getBanditModels(): array
```

Comment:
consider naming this `getBanditModelVersions()` as that is more specifically what it is doing

Reply:
✔️